### ISSUE TITLE
default_route_table_id

### ISSUE THREAD

**Issue by pressplay21** [linuxacademy/content-terraform#1]
I notice that "{aws_vpc.tf_vpc.default_route_table.id}" doesn't need a dollar sign "$" . If it's not a terraform variable, how is it getting called?
default_route_table_id = "{aws_vpc.tf_vpc.default_route_table.id}"

**Comment by derekm1215** [comment]
There is a good chance I updated on the fly and forgot to commit. You are correct, the syntax is wrong and I will fix asap.

**Comment by pressplay21** [comment]
Actually it wouldn’t work with $. I tried it myself. Maybe a bug in
terraform?

**Comment by derekm1215** [comment]
Good eye! The reason there wasn't an error otherwise is due to the fact that I ended up not referencing that route table anywhere in the scripts. Due to time constraints, I just used two public route tables. 

The syntax of the route table id itself is actually incorrect. 

The correct syntax is here: `default_route_table_id = "${aws_vpc.tf_vpc.default_route_table_id}"` It's a slightly different resource than the others used. 

More information on the default route table can be found here: https://www.terraform.io/docs/providers/aws/r/default_route_table.html

Thanks for catching that! I'll be sure to add a note to the course.
